### DISCUSSION TITLE
Exceeded daily rate limit

### DISCUSSION THREAD

**Discussion by simonj** [etsy/open-api#1220] [Q&A]
I have an issue where if one of my users uses a lot of API calls (should be 10.000 per user, right?) but it seems like it's the TOTAL request my application is sending because all my users are getting "Exceeded daily rate limit" and that is also on API calls like `getShopSections` that only require `API Key` and not the OAuth token.

So am a little unsure why this is happening.

Has anyone tried that before?

**Comment by showtroylove** [comment]
I need more information about your app to provide a more detailed answer. For instance, when you mention 'my users,' what does that entail? Have they each given your app approval using the OAuth2 authorization process? If so, describe the scopes you requested. 

1. The first step you should take is to examine the prefix of the auth token you receive for each user. They should be different, containing the user ID of the actual user to show which scope your app is currently using. If they're all the same, that's not good. 

2. Secondly, you should also capture the rate limit header information returned for each call to estimate if your instant call is accurate or if it appears to include one or more calls from one or more of your users.
